Find a functionally equivalent code:android.os.Bundle.getBoolean:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or false if
 * no mapping of the desired type exists for the given key.
 *
 * @param key a String
 * @return a boolean value
 */

@@ -1,6 +1,3 @@
 {
-    unparcel();
-    if (DEBUG)
-        Log.d(TAG, "Getting boolean in " + Integer.toHexString(System.identityHashCode(this)));
-    return getBoolean(key, false);
+    return super.getBoolean(key);
 }

